plugin/kubernetes/ns_test.go

```diff
@@ -103,7 +103,7 @@ func TestNsAddrs(t *testing.T) {
 	k.APIConn = &APIConnTest{}
 	k.localIPs = []net.IP{net.ParseIP("10.244.0.20")}
 
-	cdrs := k.nsAddrs(false, k.Zones[0])
+	cdrs := k.nsAddrs(false, false, k.Zones[0])
```

Comment:
Should add tests for when `headless` is true.

Reply:
Will do.

Reply:
I've added some tests, when you'll have time, could you please check if it's enough? :slightly_smiling_face: